client/src/components/CreateMeetup/CreateMeetup.js

```diff
@@ -1,7 +1,246 @@
-import React from "react";
+import { React, useState } from "react";
+import makeStyles from "@material-ui/styles/makeStyles";
+import {
+  Box,
+  Button,
+  Container,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+  Typography,
+} from "@material-ui/core";
+import AutoComplete from "@material-ui/lab/Autocomplete";
+import TextField from "@material-ui/core/TextField";
+import {
+  KeyboardDatePicker,
+  KeyboardTimePicker,
+  MuiPickersUtilsProvider,
+} from "@material-ui/pickers";
+import DayUtils from "@date-io/dayjs";
+
+const useStyles = makeStyles(() => ({
+  formSection: {
+    marginTop: 15,
+    marginBottom: 15,
+  },
+  formInput: {
+    width: 200,
+  },
+}));
 
 function CreateMeetup() {
-  return <>Create meetup</>;
+  const classes = useStyles();
+  const [name, setName] = useState("");
+  const [date, setDate] = useState(Date.now());
```

Comment:
Double-checking: Date.now() or new Date()?

Difference is that Date.now() returns milliseconds, while new Date() gives you a Date object.

Reply:
Good catch, I think it should be `new Date()`